docs/webhook_events.md

```diff
@@ -265,6 +265,10 @@ Each event will be submitted via HTTP POST to the user provided URL.
                 "ensemble_sync_delay": {
                     "title": "Ensemble Sync Delay",
                     "type": "integer"
+                },
+                "overwrite_output_container": {
```

Comment:
I don't think this makes clear that this setting causes us to _delete missing_ files, which is very different from just "it's okay to overwrite".